Documentation/CRDs/Object-Storage/ceph-object-store-user-crd.md

```diff
@@ -60,3 +60,88 @@ spec:
     * `user-policy`
     * `odic-provider`
     * `ratelimit`
+
+### CephObjectStoreUser Reference Secret
+
+A Kubernetes secret is created by Rook with the name, `rook-ceph-object-user-<store-name>-<user-name>` in the same namespace of cephobjectUser, where:
+
+* `store-name`: The object store name in which the user will be created. This matches the name of the objectstore CRD.
+* `user-name`: The metadata name of the cephObjectStoreUser
+
+Application pods can use the secret to configure access to the object store.
+
+```yaml
+apiVersion: v1
+data:
+  AccessKey: ***       # [1]
+  Credentials: ***     # [2]
+  Endpoint: ***        # [3]
+  SecretKey: ***       # [4]
+kind: Secret
+...
+...
+type: kubernetes.io/rook
+
+```
+
+1. `AccessKey`: User access key for ceph S3.
+2. `SecretKey`: User secret key for ceph S3.
+3. `Credentials`: Comma separated values of all access and secret keys.
```

Comment:
Let's add an explanation of the data format so users know how to interact with it.